File: backend/src/main/java/com/tencheeduard/hotelapp/entities/Account.java
package com.tencheeduard.hotelapp.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.sql.Date;

@Entity
@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor
public class Account {

    @Id
    String username;

    String firstName;

    String lastName;

    String password;

    String email;

    String phoneNumber;

    Date joinDate;

}

File: frontend/src/app/classes/entities/Account.ts
export class Account {

    username : string

    firstName: string

    lastName : string

    password : string

    email : string

    phoneNumber : string

    joinDate : Date

    constructor(username : string, firstName: string, lastName : string, password : string, email : string, phoneNumber : string, joinDate : Date)
    {
        this.username = username;
        this.firstName = firstName;
        this.lastName = lastName;
        this.password = password;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.joinDate = joinDate;
    }

}